docker-compose.yml

```diff
@@ -1,12 +1,24 @@
 services:
-  shell:
+  prod-shell:
+    image: front-end-monorepo_prod:latest
+    build:
+      context: ./
+      args:
+        - NODE_ENV=production
+        - PANOPTES_ENV=production
+        - NEXT_TELEMETRY_DISABLED=1
+        - APP_ENV=development
+    command:
+      - "/bin/sh"
```

Comment:
This runs `bin/sh` inside the `front-end-monorepo_prod` container, giving you a bash shell so that you can poke around the file system and debug the production build.

`dev-shell` does the same thing for `front-end-monorepo_dev`, which has dev dependencies installed (so you can also run tests from that shell.)